code/game/objects/items/weapons/twohanded.dm

```diff
@@ -222,6 +222,11 @@
 
 /obj/item/weapon/twohanded/dualsaber/attack(mob/target, mob/living/carbon/human/user)
 	..()
+	if(user.has_dna())
+		if(user.dna.check_mutation(HULK))
+			user << "<span class='warning'>You grip the blade too hard and accidentally close it!</span>"
+			unwield()
+			return
```

Comment:
i wasn't sure if i should put this before the parent call. i think as it is, the hulk can still get one attack off, but then the blade will close immediately after. hopefully this is fine.